stylesheets/default_stylesheet.py

```diff
@@ -0,0 +1,81 @@
+from dash import Dash, html, Input, Output, dcc, ctx, State
+import dash_bootstrap_components as dbc
+import dash_cytoscape as cyto
+import module_data 
+
+neutral_node_styling = {
```

Comment:
I really like that you put a lot of the css in its own module so it can be reused for multiple components.